code/hud/hudtarget.cpp

```diff
@@ -6541,7 +6541,7 @@ void HudGaugeOffscreen::calculatePosition(vertex* target_point, vec3d *tpos, vec
 	g3_project_vertex(eye_vertex);
 
 	if (eye_vertex->flags&PF_OVERFLOW) {
-		Int3();			//	This is unlikely to happen, but can if a clip goes through the player's eye.
+		//	This is unlikely to happen, but can if a clip goes through the player's eye.
 		Player_ai->target_objnum = -1;
```

Comment:
I'm not sure this should stay either.  Removing the player's target here doesn't make sense, unless I'm reading this wrong.

Reply:
yeah good point